core/creator/src/main/java/io/quarkus/creator/phase/augment/AugmentTask.java

```diff
@@ -177,14 +180,17 @@ public AugmentOutcome run(CurateOutcome appState, CuratedApplicationCreator ctx)
                 builder.addFinal(BytecodeTransformerBuildItem.class)
                         .addFinal(ApplicationArchivesBuildItem.class)
                         .addFinal(MainClassBuildItem.class)
-                        .addFinal(ArtifactResultBuildItem.class);
+                        .addFinal(ArtifactResultBuildItem.class)
+                        .addFinal(DockerImageResultBuildItem.class)
```

Comment:
This should not be needed as ArtifactResultBuildItem is intended to be the generic output type when extensions add additional outputs.

Reply:
It doesn't feel that ArtfiactResultBuildItem is suitable for representing a docker image.
It's something more tailored around physical files.

Reply:
DockerImageResultItem has been removed.